custom/icds_reports/models.py

```diff
@@ -121,6 +123,8 @@ class AggAwcMonthly(models.Model):
     district_map_location_name = models.TextField(blank=True, null=True)
     state_map_location_name = models.TextField(blank=True, null=True)
     month = models.DateField(blank=True, null=True)
+    aww_name = models.DateField(blank=True, null=True)
+    contact_phone_number = models.DateField(blank=True, null=True)
```

Comment:
@czue @sravfeyn  Do you know why ```aww_name``` and ```contact_phone_numer``` are ```DateField```

Reply:
I do not, nor can I fathom an explanation. Guessing it's a mistake?

Reply:
@czue I also guessing it's a mistake. I will fix this when I will send PR with changes to the Monthly Register report.

Reply:
that'd be great, thanks!